src/main/java/com/enderio/core/client/render/RenderUtil.java

```diff
@@ -376,6 +376,24 @@ public static void addVerticesToTessellator(List<Vertex> vertices, Tessellator t
         }
     }
 
+    public static void addVerticesToTessellator(Vertex[] vertices, Tessellator tes) {
+        for (Vertex v : vertices) {
```

Comment:
nit: If you're trying to reduce allocations use the c style iteration vs foreach which allocates an iterator